testing/cloud_layouts/Static/infra.tpl.tf

```diff
@@ -99,11 +99,11 @@ resource "openstack_compute_instance_v2" "system" {
 }
 
 resource "openstack_compute_floatingip_v2" "master" {
-  pool = data.openstack_networking_network_v2.shared.name
+  pool = data.openstack_networking_network_v2.external.name
```

Comment:
I did not get this change. Why is it required to use ips from the external network? Their number in our opestack cloud is limited.

Reply:
this is only for simplify access to cluster for debug purposes.